agent/src/beerocks/slave/ap_manager_thread.cpp

```diff
@@ -777,9 +777,11 @@ bool ap_manager_thread::handle_cmdu(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_
             return true;
         }
 
-        LOG(DEBUG) << "CLIENT_ALLOW: mac = " << sta_mac << ", bssid = " << bssid;
+        remove_client_from_disallowed_list(m_disallowed_clients, request->mac(), request->bssid());
 
+        LOG(DEBUG) << "CLIENT_ALLOW: mac = " << sta_mac << ", bssid = " << bssid;
```

Comment:
This log should be moved to `remove_client_from_dissallowed_list()`

Reply:
no,
1. since there is chance that remove_client_from_dissallowed_list() doesn't find the client on the list.
2. ALLOW message not actual sent in remove_client_from_dissallowed_list() so this log could be misleading.